rllib/env/tests/test_multi_agent_env.py

```diff
@@ -329,19 +329,25 @@ def compute_actions(
             def get_initial_state(self):
                 return [{}]  # empty dict
 
+            def is_recurrent(self):
```

Comment:
We need to add a todo somewhere that we start to properly infer this.
If a model has an initial state, it's recurrent, otherwise it's not. Right?

Reply:
only for torch and tensorflow policies do we have this automatically detected. not for policies that inherit from the base policy class.

Reply:
.. and that's not ok, right?

Reply:
yeah I'm adding the todo

Reply:
since it's a unittest it's ok to hardcode. but the TODO is fine. No need to recommit for this.

Reply:
What do you mean? There is no TODO atm. 
I am required to add such things as todos everytime I discover something, what makes this case different?

Reply:
I am saying the hardcoding in unittests is fine in general which is the case here. @avnishn has added a TODO like you suggested.

Reply:
Got it!